gnocchi/tests/functional/gabbits/aggregates-with-resources.yaml

```diff
@@ -0,0 +1,190 @@
+      desc: for later use
+      POST: /v1/archive_policy
+      request_headers:
+        # User admin
+        authorization: "basic YWRtaW46"
+      data:
+        name: low
+        definition:
+          - granularity: 1 second
+          - granularity: 300 seconds
+      status: 201
+
+    - name: create another archive policy
+      desc: for later use
+      POST: /v1/archive_policy
+      request_headers:
+        # User admin
+        authorization: "basic YWRtaW46"
+      data:
+        name: unrelated
+        definition:
+          - granularity: 5 second
+      status: 201
+
+    - name: create resource 1
+      POST: /v1/resource/generic
+      data:
+        id: 4ed9c196-4c9f-4ba8-a5be-c9a71a82aac4
+        user_id: 6c865dd0-7945-4e08-8b27-d0d7f1c2b667
+        project_id: c7f32f1f-c5ef-427a-8ecd-915b219c66e8
+        metrics:
+          cpu.util:
+            archive_policy_name: low
+      status: 201
+
+    - name: post cpuutil measures 1
+      POST: /v1/resource/generic/4ed9c196-4c9f-4ba8-a5be-c9a71a82aac4/metric/cpu.util/measures
+      data:
+        - timestamp: "2015-03-06T14:33:57"
+          value: 43.1
+        - timestamp: "2015-03-06T14:34:12"
+          value: 12
+      status: 202
+
+    - name: create resource 2
+      POST: /v1/resource/generic
+      data:
+        id: 1447CD7E-48A6-4C50-A991-6677CC0D00E6
+        user_id: 6c865dd0-7945-4e08-8b27-d0d7f1c2b667
+        project_id: c7f32f1f-c5ef-427a-8ecd-915b219c66e8
+        metrics:
+          cpu.util:
+            archive_policy_name: low
+      status: 201
+
+    - name: post cpuutil measures 2
+      POST: /v1/resource/generic/1447CD7E-48A6-4C50-A991-6677CC0D00E6/metric/cpu.util/measures
+      data:
+        - timestamp: "2015-03-06T14:33:57"
+          value: 23
+        - timestamp: "2015-03-06T14:34:12"
+          value: 8
+      status: 202
+
+    - name: create resource 3
+      POST: /v1/resource/generic
+      data:
+        id: 33333BC5-5948-4F29-B7DF-7DE607660452
+        user_id: 6c865dd0-7945-4e08-8b27-d0d7f1c2b667
+        project_id: ee4cfc41-1cdc-4d2f-9a08-f94111d80171
+        metrics:
+          cpu.util:
+            archive_policy_name: low
+      status: 201
+
+    - name: post cpuutil measures 3
+      POST: /v1/resource/generic/33333BC5-5948-4F29-B7DF-7DE607660452/metric/cpu.util/measures
+      data:
+        - timestamp: "2015-03-06T14:33:57"
+          value: 230
+        - timestamp: "2015-03-06T14:34:12"
+          value: 45.41
+      status: 202
+
+    - name: create resource 4
+      POST: /v1/resource/generic
+      data:
+        id: b1409ec6-3909-4b37-bbff-f9a5448fe328
+        user_id: 70b5b732-9d81-4dfb-a8a1-a424ef3eae6b
+        project_id: ee4cfc41-1cdc-4d2f-9a08-f94111d80171
+        metrics:
+          cpu.util:
+            archive_policy_name: low
+      status: 201
+
+    - name: post cpuutil measures 4
+      POST: /v1/resource/generic/33333BC5-5948-4F29-B7DF-7DE607660452/metric/cpu.util/measures
```

Comment:
still posting to resource 3 and not resource 4

Reply:
lol, the test that use it is also true when no measure are inside metric4

Reply:
ooohhh. metric4 is there to prove it's not being picked up by search query. just noticed it was a different user_id